api_tests/tests/reasonable_adjustment_flag/test_error_cases.py

```diff
@@ -350,6 +350,44 @@ def test_list_invalid_header_put(self):
         assert_that(expected_response['error']).is_equal_to_ignoring_case(actual_response['error'])
         assert_that(expected_response['error_description']).is_equal_to_ignoring_case(actual_response['error_description'])
 
+    @pytest.mark.errors
+    @pytest.mark.integration
+    @pytest.mark.usefixtures('get_token_internal_dev')
+    def test_removerarecord_invalid_header_post(self):
+        # Given
+        expected_status_code=400
+        expected_response={
+            "error": "invalid header",
+            "error_description": "if-match is missing or invalid"
+        }
+
+        # When
+        response = requests.post(
+            url=config.REASONABLE_ADJUSTMENTS_REMOVE_RA_RECORD,
+            params={
```

Comment:
`$removerarecord` doesn't need query parameters. I removed proxy `<Condition>` checks on my PR but it's still blocked. Can you please remove all the query params related to this endpoint just to prevent merge conflicts.